src/main/java/gighub/worketserver/global/exception/PasscodeErrorCode.java

```diff
@@ -7,10 +7,11 @@
 @Getter
 @AllArgsConstructor
 public enum PasscodeErrorCode implements ErrorCode {
-  PASSCODE_EMPTY(HttpStatus.FORBIDDEN, "AUTH_3001", "패스코드가 비어 있습니다.");
+
+  INVALID_PASSCODE(HttpStatus.UNAUTHORIZED, "AUTH_3002", "잘못된 패스코드입니다.");
```

Comment:
httpStatus 도 403으로 줘야하지 않을까요???

```suggestion
  INVALID_PASSCODE(HttpStatus.FORBIDDEN, "AUTH_3002", "잘못된 패스코드입니다.");
```

Reply:
아 맞네요 ㄳ합니다